core/object.cpp

```diff
@@ -2036,7 +2036,8 @@ void ObjectDB::remove_instance(Object *p_object) {
 
 	rw_lock.write_unlock();
 }
-Object *ObjectDB::get_instance(ObjectID p_instance_id) {
+
+Object *ObjectDB::_get_instance(ObjectID p_instance_id) {
```

Comment:
As in KoBeWi's PR, I don't think a separate `_get_instance` is needed. The templated version should be able to call the `get_instance` as is.

Reply:
Will try this. :+1:

Reply:
No doesn't seem to work. Seems to need this version for e.g.:
```
	Object *areaGodoObject = ObjectDB::get_instance(event.event_callback_id);
```
to compile.

Reply:
You don't need to eliminate the second version! I'm proposing to rename `_get_instance(ObjectID p_instance_id)` as you have it right now to `get_instance(ObjectID p_instance_id)`, and expose it into the `public` space. Then, you'll just have two functions!

Reply:
Yeah I finally figured out what the other PR was doing, I think mine was historical, then I had some out of date compilation intermediates errors to sort out, and finally I think it's the same. :grin: